public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -22,6 +22,7 @@ interface Props {
   pinned?: boolean;
   styles: LogRowStyles;
   mouseIsOver: boolean;
+  onBlur?: () => void;
```

Comment:
Same as before

Reply:
Is there another comment? I can't find it.